### ISSUE TITLE
As a consumer, I need to provide API keys to the library so that I can access my ClickUp account data.

### ISSUE THREAD

**Issue by fenterbug** [fenterbug/ClickUp.Net#2]

**Comment by dochoss** [comment]
Provided in the main class constructor?  I can put together a quick implementation tomorrow, I think.

**Comment by fenterbug** [comment]
I've thought about that, as well as an appsettings.json file and a couple of other things. Which is the _right_ way? I don't know. It's probably the constructor approach like you suggest. I say go for it. Get's me out of my analysis paralysis.

**Comment by fenterbug** [comment]
Of course, it might be influenced by whatever it takes for the OAuth authentication. (??)

**Comment by dochoss** [comment]
Depends on if you want it set at runtime or build time. To use the class in code, you'd want it available at runtime. I'll put together an implementation tomorrow and get things moving 😝
________________________________
